src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1838,17 +1836,14 @@ FixedwingPositionControl::tecs_update_pitch_throttle(const hrt_abstime &now, flo
 
 	if (_vehicle_status.engine_failure) {
 		/* Force the slow downwards spiral */
-		pitch_min_rad = M_DEG_TO_RAD_F * -1.0f;
-		pitch_max_rad = M_DEG_TO_RAD_F * 5.0f;
+		pitch_min_rad = radians(-1.0f);
```

Comment:
Note, we should revisit engine failure

Reply:
Do you know of anybody that uses it actually?

Reply:
I did a few years ago. I'd bet people don't even know it exists.